internal/web/app/application.go

```diff
@@ -32,18 +35,67 @@ func New(params Params) *Application {
 
 func (a *Application) Run() error {
 	conf, err := config.UnmarshalFromYamlConfiguration(a.Params.configFilePath)
+	setupLogging(conf)
```

Comment:
This should probably come after the error check?

Reply:
It can't, because then we can't log the error.

setupLogging handles nil as conf and uses ECS logging if nil.